rust-sgx-sdk/version

```diff
@@ -1 +1 @@
-565960cd7b4b36d1188459d75652619971c43f7e
+e8a9fc22939befa27ff67f5509b2c2dfe8499945
```

Comment:
Can you redo this change, or, if you want to update, also update the cargo.lock accordingly?